public/kibi_timeline.js

```diff
@@ -226,7 +227,10 @@ define(function (require) {
           if ($scope.groups) {
             initGroups();
             // do not use newValue as it does not have searchSource as we filtered it out
-            _.each($scope.groups, initSingleGroup);
+            _.each($scope.groups, (group, index) => {
+              requestQueue.markAllRequestsWithSourceIdAsActive(group.searchSource._id);
```

Comment:
please check here 
so we do not have the case that you marking the old once as active 
and at the same time the controller is creating the new set of searchSources 
If this would be the case then simply marking as inactive and forget about them should be ok